eng/pipelines/aggregate-reports.yml

```diff
@@ -1,3 +1,13 @@
+trigger: none
+
+pr:
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - eng/pipelines/aggregate-reports.yml
```

Comment:
I suspect you might need to go and undo the override to disable the PR trigger in these pipelines. I had to do that for the python one earlier.